jbi/models.py

```diff
@@ -23,6 +23,18 @@
 JIRA_HOSTNAMES = ("jira", "atlassian")
 
 
+class ActionParams(BaseModel):
+    """Params passed to Action step functions"""
+
+    jira_project_key: str
+    steps: Optional[dict[str, list[str]]]
+    jira_components: list[str] = []
+    labels_brackets: Optional[str] = Field("no", enum=["yes", "no", "both"])
```

Comment:
If there's a default value, should it be optional? Why not like other fields?

Reply:
Ahh yeah good catch. [db44bc5](https://github.com/mozilla/jira-bugzilla-integration/pull/591/commits/db44bc531021f9b16d032941c1f2f590120fd78e)